CompositeUi/src/views/component/UserCard.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of the Kreta package.
+ *
+ * (c) Beñat Espiña <benatespina@gmail.com>
+ * (c) Gorka Laucirica <gorka.lauzirika@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import React from 'react';
+
+import CardExtended from './CardExtended';
+import UserThumbnail from './UserThumbnail';
+
+class UserCard extends React.Component {
+  static propTypes = {
+    user: React.PropTypes.object.isRequired
+  };
+
+  name() {
+    const {user} = this.props;
+
+    if (user.first_name && user.last_name) {
+      return `${user.first_name} ${user.last_name}`;
+    }
+
+    return user.email;
```

Comment:
Maybe we can check for username here if name-surname are not set before using email. Maybe the logic is more complex, because if the username is used in the title in the subtitle we should not show anything